dll/cpl/input/lang/cs-CZ.rc

```diff
@@ -10,10 +13,10 @@ STYLE DS_SHELLFONT | WS_CHILD | WS_DISABLED | WS_CAPTION
 CAPTION "Nastavení"
 FONT 8, "MS Shell Dlg"
 BEGIN
-    GROUPBOX "Default input &language", -1, 5, 5, 246, 55
-    LTEXT "Select one of the installed input languages to use when your computer starts.", -1, 15, 15, 230, 23
+    GROUPBOX "Výchozí vstupní &jazyk", -1, 5, 5, 246, 55
+    LTEXT "Vyberte jeden z nainstalovaných jazyků, který se bude používat po spuštění Vašeho počítače.", -1, 15, 15, 230, 23
     COMBOBOX IDC_DEFAULT_LANGUAGE, 15, 40, 230, 300, CBS_HASSTRINGS | CBS_AUTOHSCROLL | CBS_DROPDOWNLIST | WS_VSCROLL | WS_TABSTOP
-    GROUPBOX "&Installed services", -1, 5, 65, 246, 125
+    GROUPBOX "Na&instalované služby", -1, 5, 65, 246, 125
```

Comment:
Well, then let's place it at the &s for ros as well. I checked: the &s isn't used anywhere else in the dlg.
```suggestion
    GROUPBOX "Nainstalované &služby", -1, 5, 65, 246, 125
```

Reply:
I've also changed other strings according to the screenshot.

Reply:
I checked them regarding accels. LGTM. Resolved, and final approval hereby.